charts/gcloud-sqlproxy/Chart.yaml

```diff
@@ -16,4 +16,4 @@ maintainers:
 name: gcloud-sqlproxy
 sources:
 - https://github.com/broadinstitute/datarepo-helm
-version: 0.19.9
+version: 0.19.10
```

Comment:
@snf2ye -- I'd especially like your confirmation that I've handled the chart bump correctly.

Reply:
Yes, this looks correct. Thanks for remembering to bump the version 😅